packaging/nnstreamer.spec

```diff
@@ -229,6 +232,9 @@ BuildConflicts: libarmcl-release
 %if 0%{?edgetpu_support}
 BuildRequires:	pkgconfig(edgetpu)
 %endif
+%if 0%{?aitt_support}
+BuildRequires:  aitt-devel
+%endif
```

Comment:
Add a comment that this is only required for unit testing.

Reply:
Or, check nnstreamer_edge_support feature. (edgesrc/edgesink/tensor_query* all have dependency to nns-edge library.)